applications/GeoMechanicsApplication/custom_elements/README.md

```diff
@@ -0,0 +1,105 @@
+- $\lambda^s$   = thermal conductivity solid matrix $\mathrm{[W/m ^{\circ}C]}$
+
+In the absence of ground water flow, these equations are simplified to,
+
+$$ \left(n S \rho^w c^w + (1- n) \rho^s c^s \right) \frac{\partial T}{\partial t} = \frac{\partial}{\partial x_i} \left( D_{ij} \frac{\partial T}{\partial x_j} \right) \qquad \text{on} \quad \Omega $$
+
+$$ D_{ij}= nS \lambda^w \delta_{ij} + \left(1-n\right) \lambda_{ij}^s $$
+
+## Boundary Conditions
+
+### Dirichlet boundary condition
+$$ T = \overline T \qquad \text{on} \quad \Gamma_{1}^T $$
+
+where $\overline T$ $\mathrm{\left[ ^{\circ}C \right]}$ is a prescribed temperature 
+
+### Neumann Boundary Condition
+$$ D_{ij} \frac{\partial T}{\partial x_j} n_i = \overline{f} \qquad \text{on} \quad \Gamma_{2}^T $$
+
+where $\overline f$ $\mathrm{\left[ W/m^2 \right]}$ is a prescribed conductive heat flux.
+
+### Robin bounday condition
+$$ D_{ij} \frac{\partial T}{\partial x_j} n_i = \overline{g} -  \rho^w c^w q_n T \qquad \text{on} \quad \Gamma_{3}^T $$
+
+where $\overline g$ $\mathrm{\left[ W/m^2 \right]}$ is a prescribed convective-conductive heat flux.
+
+
+## Derived Properties
+
+The density of the bulk material $\rho$ $\mathrm{\left[ kg/m^3 \right]}$ is calculated as,
+
+$$ \rho = n S \rho^w + \left( 1 - n \right) \rho^s $$
+
+And the heat capacity of the bulk material $C$ $\mathrm{\left[ J/m{^3 \circ}C \right]}$ is:
+
+$$ C = n S \rho^w c^w + \left( 1 - n \right) \rho^s c^s $$
+
+The thermal conductivity of the bulk material $\lambda$ $\mathrm{\left[ W/m ^{\circ}C \right]}$
+
+$$ \lambda = n S \lambda^w + \left( 1 - n \right) \lambda^s $$
+
+dynamic viscosity  $\mu$ $[\mathrm {Pas}$] of pure water as a function of temperature is given by:
+
+$$ \mu = 2.4318 \cdot 10^{-5} \cdot 10^{{247.8} / {\left(T+133.0\right]}} $$
+
+density of water $\rho^w$ $[\mathrm {kg/m^3}]$ relates to temperature and Diersch proposes a six order Taylor expansion which is approximated here by:
+
+$$ \rho^w = 9.998396 \cdot 10^2 + 6.764771  \cdot 10^{-2} \cdot T - 8.993699  \cdot 10^{-3} \cdot T^2 + 9.143518 \cdot 10^{-5} \cdot T^3 - 8.907391 \cdot 10^{-7} \cdot T^4 + 5.291959  \cdot 10^{-9} \cdot T^5 - 1.359813  \cdot 10^{-11} \cdot T^6 $$
+
+
+
+## Finite Element Formulation
+
+Kratos solves the equations based on incremental method. In The fram of Generelized Newmark method (GN11), the fully implicit incremental temperature formulation read as,
+
+$$ \left(\frac{1}{\theta \Delta t} \boldsymbol{S} + \boldsymbol{A} + \boldsymbol{H} + \boldsymbol{W}^l  \right) \boldsymbol{\Delta T} = \left( \frac{1}{\theta} - 1 \right) \boldsymbol{S} \frac{dT^n}{dt} - \left(\boldsymbol{A} + \boldsymbol{H} + \boldsymbol{W}^l \right) \boldsymbol{T}^{n} + \left( \boldsymbol{V} + \boldsymbol{W}^r \right) $$
+
+***Compressibility matrix***
+
+$$ \boldsymbol{S} = \sum_e \int_{\Omega^e} \left( n S \rho^w c^w + \left(1-n\right) \rho^s c^s \right)^{n+1} \boldsymbol{N}^T  \boldsymbol{N} d \Omega $$
```

Comment:
the combination of summation and integration here and in all formulas below looks strange to me. The summation over elements or element integration points together with multiplication with element or i.p. volumes replaces the integral.